pyodide_build/tests/test_cli_xbuildenv.py

```diff
@@ -25,6 +26,18 @@ def mock_pyodide_lock() -> PyodideLockSpec:
     )
 
 
+@pytest.fixture()
+def is_valid_json():
+    def _is_valid_json(json_str):
```

Comment:
Question: Why is it defined as a fixture? Doesn't calling `is_valid_json` work?

Reply:
Oh, I was using it in the test invocation, which meant that it needed to be a fixture for `pytest` to recognise it. But we don't need that, necessarily. In a9f9a61ee3936f05f7018f6491f934de09950a9d, I've removed the fixture and called it directly, as you suggest.